src/pages/Login/index.tsx

```diff
@@ -25,12 +25,11 @@ const Login: React.FC = () => {
     if (userPubkey) {
       // Only need to trigger signatures for notify if none were issued for chat
       const notifyConditionsPass = Boolean(
-        (!uiEnabled.chat && uiEnabled.notify) || notifyRegisteredKey
+        !uiEnabled.chat && uiEnabled.notify && notifyRegisteredKey
       )
 
       if (notifyConditionsPass) {
         nav(path)
-        // Else if signature is required.
```

Comment:
The comment I removed didn't add any value in understanding the logic of the code. Comments are useful when they explain complex logic, but in this case, the comment was merely repeating what was obvious in the following line. Good code should be self-explanatory and clean, minimizing the need for comments. If we find ourselves relying heavily on comments, it might indicate that the code isn't clear enough. In this instance, the code is straightforward and easy to understand without the comment.